### DISCUSSION TITLE
Backporting of PRs

### DISCUSSION THREAD

**Discussion by mcollina** [nodejs/undici#3526] [General]
How do we want to backport PRs to v6.x to be included in Node.js v22 and v20?

@KhafraDev @Uzlopak I think you have done a few bugfixes, could you cherry-pick the commits into the v6.x branch?

Please add the `backport v6.x` label to automatically backport via actions (let's see if it works)

**Comment by metcoder95** [comment]
Beyond the bugfixes, do we know what else do we want to backport?

**Reply by mcollina** [reply]
I can see we might want to backport certain perf improvements or new features (like caching).

**Reply by metcoder95** [reply]
I'll start backporting some of the interceptors this weekend

**Comment by KhafraDev** [comment]
The only bug fix I can think of is https://github.com/nodejs/undici/commit/e96178f1f2aa468d23b1af29b81f1f6278a357c5, but since WebSocket internals were rewritten I can't rebase easily. I'll work on fixing it in a separate PR.

**Comment by Uzlopak** [comment]
I will backport my PRs today

**Reply by Uzlopak** [reply]
The backporting label seems to be unable to handle the changes properly?

**Reply by metcoder95** [reply]
As soon as it detects there are already overlapping changes on top of the commit (PR) you want to backport, and it can have conflicts, it fails. Tried with some of your PRs without success.
The ones that are _cleaner_ goes smoothly

**Comment by Uzlopak** [comment]
I really want to backport the PRs but the suggested way is not working properly. Is it acceptable to "manually" backport?

**Reply by Uzlopak** [reply]
@mcollina

**Reply by mcollina** [reply]
yes of course!
